notebook/services/config/manager.py

```diff
@@ -5,13 +5,46 @@
 
 import os.path
 
-from traitlets.config.manager import BaseJSONConfigManager
-from jupyter_core.paths import jupyter_config_dir
-from traitlets import Unicode
+from traitlets.config.manager import BaseJSONConfigManager, recursive_update
+from jupyter_core.paths import jupyter_config_dir, jupyter_config_path
+from traitlets import Unicode, Instance, List
+from traitlets.config import LoggingConfigurable
 
-class ConfigManager(BaseJSONConfigManager):
+
+class ConfigManager(LoggingConfigurable):
     """Config Manager used for storing notebook frontend config"""
-    
-    config_dir = Unicode(config=True)
-    def _config_dir_default(self):
+
+    # Public API
+
+    def get(self, section_name):
+        """Get the config from all config sections."""
+        config = {}
+        for p in self.read_config_path:
+            cm = BaseJSONConfigManager(config_dir=p)
+            recursive_update(config, cm.get(section_name))
+        return config
+            
+    def set(self, section_name, data):
+        """Set the config only to the user's config."""
+        self._write_config_manager.set(section_name, data)
```

Comment:
This function is non existent and is causing the tests to fail, what was intended by this line?  I couldn't find the function defined in traitlets or notebook...